periodic-table/index.md

```diff
@@ -0,0 +1,16 @@
+---
+title: PERIODIC TABLE 
+uuid: periodic-table
+prerequisites:
+  - 
+learning_objectives: 
+  - 
```

Comment:
do boundary nodes need learning objectives?

Reply:
I left it like that because I figured in the future someone might want to come in and populate the node.